tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -126,3 +138,23 @@ Verify Cuda Builds Are Completed
         END
         Should Be Equal As Strings    ${pre}[3]    Complete
     END
+
+Verify Link And Run Commands On Prometheus Pod For Jupyterhub And Rhods-Dashboard URLs
```

Comment:
- See comment above about renaming this keywod
- [Documentation] Verifies authentication is required to access blackbox exporter. To do so, runs the curl command from the prometheus container trying to access a blacbox-exporter target. The test fails if the response is not a prompt to log in with OpenShift